service/src/main/java/bio/terra/landingzone/stairway/flight/create/CreateAzureLandingZoneDbRecordStep.java

```diff
@@ -29,36 +29,20 @@ public CreateAzureLandingZoneDbRecordStep(LandingZoneDao landingZoneDao) {
   public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
     final FlightMap inputMap = context.getInputParameters();
     FlightUtils.validateRequiredEntries(
-        inputMap, LandingZoneFlightMapKeys.LANDING_ZONE_CREATE_PARAMS);
+        inputMap,
+        LandingZoneFlightMapKeys.LANDING_ZONE_CREATE_PARAMS,
+        LandingZoneFlightMapKeys.BILLING_PROFILE,
+        LandingZoneFlightMapKeys.LANDING_ZONE_ID);
 
     var requestedExternalLandingZoneResource =
         inputMap.get(LandingZoneFlightMapKeys.LANDING_ZONE_CREATE_PARAMS, LandingZoneRequest.class);
-    var landingZoneTarget = requestedExternalLandingZoneResource.landingZoneTarget();
```

Comment:
I am now generating the landing zone ID at flight creation time, so we can treat it as a required input parameter.